### PR TITLE
Middle section of the mobile website

### PR DESCRIPTION
**Pull request by aimalamiri**
Designed two cards to show recent projects one for the featured project and one for the other projects, also designed the about me section for mobile layout.
The design for the button with 3 basic behaviors hover, active, and disabled is also added.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -89,6 +104,171 @@ header {
...
+  background: #ff6b00;
+  color: #fff;
+}
+
+.button:hover {
+  transition: all 0.2s ease-in-out;
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
```
Comment (nicupop729) [inline styles.css]: [REQUIRED] While you have done a great job implementing the style of the buttons, you missed the style when the buttons are pressed.

- [ ] Please make sure to implement the style for the buttons when active, accordingly with the following figma design:
image

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @aimalamiri, 

You are doing very well!
There are very small issues that you still need to work on to go to the next project but you are almost there!



 ## To Highlight
* Correct GitHubFlow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 
* Clean codebase ✔️ 

### Required Changes ♻️

- [x] Pull Request should have a short summary describing introduced changes in more detail. "The design of the middle section for mobile is finished" cannot be named as a short summary describing introduced changes in more detail. You should name exactly the sections you worked on and maybe explain what these sections contain.
- [x] `node_modules` folder should be in the `.gitignore` Please make sure to add it there.
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (aimalamiri) [reply styles.css]: Hi @nicupop729 thanks for the review, I'm done with the required changes.

File: index.html [hunk 2]
```diff
@@ -66,6 +66,170 @@ <h2 class="header-subtitle text-white font-inter">I’m a Software Developer</h2
...
+                    <div>
+                        <h3 class="project-title text-white font-crete-round">Profesional Art Printing Data</h3>
+                        <p class="project-description text-white mt-12">A daily selection of privately personalized
+                            reads; no
+                            accounts or sign-ups required. has been the industry's standard</p>
+                        <ul class="tags">
+                            <li class="tag text-white dark-tag mb-6">HTML</li>
+                            <li class="tag text-white dark-tag mb-6">Bootstrap</li>
+                            <li class="tag text-white dark-tag mb-6">Ruby</li>
+                        </ul>
```
Comment (MrBrN197) [inline index.html:132]: - [x] [OPTIONAL] These technologies tags are done well. although you should remember to use the correct font from the design I notice they don't have the correct font set.

image

image

**Review by MrBrN197 — APPROVED** [review]
## Hi, @aimalamiri  👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (aimalamiri) [reply index.html:132]: Thanks @MrBrN197, I fixed the font issue as well.

**Pull request merged**